charts/opentelemetry-collector/README.md

```diff
@@ -107,6 +109,44 @@ agentCollector:
     mountPropagation: HostToContainer
 ```
 
+### Configuration for Kubernetes container logs
+
+The collector is preconfigured to receive logs sent to standard output by Kubernetes containers.
+
+This feature needs agent collector to be deployed, which means it will not work if only standalone collector is enabled.
+
+The feature can be disabled by setting the `agentCollector.containerLogs.enabled` property to `false`.
```

Comment:
For metrics and traces pipelines, we set them to null to disable, would it be possible to keep this pattern?

```yaml
config:
  service:
    pipelines:
      logs: null
```

Reply:
It seems to me these are two different things: disabling the whole logs pipeline vs. disabling container logs collection. You might want to disable container logs collection but keep the logs pipeline for other uses?

Reply:
Unless the container logs collection should be in its own pipeline? As opposed to a general "logs" pipeline.

Reply:
a `logs` pipeline does seem like the right thing